.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,118 @@
+---
+name: Intergration test
+
+on:
+  pull_request
+
+jobs:
+  minikube-ci:
+    name: Intergration test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Start minikube
+        uses: medyagh/setup-minikube@latest
+
+      - name: Install Dependencies
+        run: |
+          # Install yq
+          wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq
+          yq --version
+
+      - name: Build csi-snapshot-metadata container image
+        run: |
+          # make build-csi-snapshot-metadata
+          # minikube image build -f ./cmd/csi-snapshot-metadata/Dockerfile -t gcr.io/k8s-staging-sig-storage/csi-snapshot-metadata:test .
+
+      - name: Deploy snapshot-controller
+        run: |
+          ./scripts/deploy-snapshot-controller.sh deploy
+
+      - name: Deploy csi-hostpath-driver
+        run: |
+          kubectl apply -f ./client/config/crd/cbt.storage.k8s.io_snapshotmetadataservices.yaml
+
+          git clone https://github.com/rakshith-r/csi-driver-host-path.git ~/csi-driver-host-path
```

Comment:
I'll switch this with official repo once https://github.com/kubernetes-csi/csi-driver-host-path/pull/575/files goes in.